packages/cli/src/lib/implementation/core-config-options.ts

```diff
@@ -1,18 +1,34 @@
 import { Options } from 'yargs';
-import { ArgsCliObj } from './model';
+import { CoreConfigCliOptions } from './model';
 
-export function yargsCoreConfigOptionsDefinition(): Record<
-  keyof ArgsCliObj,
-  Options
-> {
+type ArgNames = keyof CoreConfigCliOptions;
+export function yargsCoreConfigOptionsDefinition(): Record<ArgNames, Options> {
   return {
-    format: {
+    // persist
+    'persist.outputDir': {
+      describe: 'Directory for the produced reports',
+      type: 'string',
+    },
+    'persist.format': {
       describe: 'Format of the report output. e.g. `md`, `json`, `stdout`',
       type: 'array',
     },
-    apiKey: {
-      describe: 'apiKey for the portal',
+    // update
+    'upload.organization': {
+      describe: 'organization name in your CVS',
+      type: 'string',
+    },
+    'upload.project': {
+      describe: 'project name in your CVS',
       type: 'string',
     },
```

Comment:
- What's CSV in this context? :raised_eyebrow: 
- It's not the name, but the slug, that could be misleading.
- The description formatting is inconsistent, `--upload.organization` and `--upload.project` descriptions aren't capitalized like the others.

```suggestion
    // upload
    'upload.organization': {
      describe: 'Organization slug from portal',
      type: 'string',
    },
    'upload.project': {
      describe: 'Project slug from portal',
      type: 'string',
    },
```